taxi/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from taxi.views import index
+
+
+app_name = "taxi"
+
+urlpatterns = [
+    path("", index, name="index"),
+]
```

Comment:
Ensure that the file ends with a single blank line, as required by the checklist.